core/src/test/java/org/apache/jsieve/HeaderTest.java

```diff
@@ -57,6 +57,21 @@ public void testHeaderIsTrue() {
     /**
      * Test for Test 'header'
      */
+    @Test(expected = ThrowTestException.TestException.class)
+    public void testFoldedEncodedHeader() throws Exception {
```

Comment:
I don't really want to change this, it's a personal contribution...

If I sanitise these tests (which are ugly) I'll do it as part of my working time, with agreement from @rouazana  or discard the PR.